tests/pdk/finfet_pdk/test_benchmark_place.py

```diff
@@ -126,6 +126,38 @@ def test_b3(name, params):
         x0, y0, x1, y1 = placement['modules'][0]['bbox']
         area_new = (x1-x0)*(y1-y0)
     print(f'\n{name}: AREA={area_new/1e8:0.2f} HPWL={hpwl_new/1e4:0.2f} TIME={elapsed_time:0.2f}')
-    if cleanup:
+    if CLEANUP:
+        shutil.rmtree(run_dir)
+        shutil.rmtree(ckt_dir)
+
+
+@pytest.mark.skipif(not BENCHMARK, reason="Exclude from CI")
+@pytest.mark.parametrize(("name", "params"), PARAMS)
+def test_b4(name, params):
```

Comment:
@srini229 , fyi. ILP is taking quite long even for 8 blocks.

Reply:
Thank you brining it up. I had noticed it while integrating the ILP placer and disabled this test in CI. I'll figure it out and enable the test.